pkg/kubectl-argo-rollouts/cmd/status/status.go

```diff
@@ -0,0 +1,122 @@
+package status
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/argoproj/argo-rollouts/pkg/kubectl-argo-rollouts/info"
+	"github.com/argoproj/argo-rollouts/pkg/kubectl-argo-rollouts/options"
+	"github.com/argoproj/argo-rollouts/pkg/kubectl-argo-rollouts/viewcontroller"
+	"github.com/spf13/cobra"
+)
+
+const (
+	statusLong = `Watch rollout until it finishes or the timeout is exceeded. Returns success if
+the rollout is healthy upon completion and an error otherwise.`
+	statusExample = `
+	# Watch the rollout until it succeeds
+	%[1]s status gestbook
```

Comment:
```suggestion
	%[1]s status guestbook
```
(I haven't contributed my spell checker to Argo yet, but it'd scream if this merged as is)